ddtrace/span.py

```diff
@@ -58,18 +62,19 @@ class Span(object):
 
     def __init__(
         self,
-        tracer,
-        name,
-        service=None,
-        resource=None,
-        span_type=None,
-        trace_id=None,
-        span_id=None,
-        parent_id=None,
-        start=None,
-        context=None,
-        _check_pid=True,
+        tracer,  # type: Tracer
```

Comment:
```suggestion
        tracer,  # type: Optional[Tracer]
```
technically (we use `None` in tests) but this should go away in favour of a callback API IMO